dislib/trees/data.py

```diff
@@ -1,5 +1,4 @@
 import tempfile
```

Comment:
please consider exposing only objects that are necessary in dislib/regression/__init__.py 
(currently all of them are exposed)